src/SprykerEco/Zed/Optivo/Business/OptivoFacadeInterface.php

```diff
@@ -1,45 +1,84 @@
 <?php
 
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
 namespace SprykerEco\Zed\Optivo\Business;
 
 use Generated\Shared\Transfer\MailTransfer;
-use Generated\Shared\Transfer\OptivoSubscribeRequestTransfer;
-use Generated\Shared\Transfer\OptivoTransactionalMailRequestTransfer;
-use Generated\Shared\Transfer\OptivoUnsubscribeRequestTransfer;
 
 interface OptivoFacadeInterface
 {
     /**
      * Specification:
-     * - Receives the fully configured MailTransfer
-     * - Sends the mail
+     *  - This method is used by OptivoNewOrderPlugin Oms command for sending data to Optivo API
```

Comment:
Please add more meaningful specification like: sends request, saves to DB, updates something and so on.

Reply:
Could you, please, show en example ?